From 9eb3cc5c51d846c8c8b750a4eb55545d7b5fea6c Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Wed, 23 Apr 2014 15:41:26 -0400
Subject: [PATCH] use LDLIBS for linker flags

When you put -lfoo into the dependency line of make, it forces it to
search /lib and /usr/lib for files to link against.  This can cause
problems when trying to cross-compile or build for different ABIs.
Use the standard LDLIBS variable instead.

URL: https://bugs.gentoo.org/505760
Reported-by: Georgi Georgiev <chutzimir@gmail.com>
Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f3838b4..f2e9a6e 100644
--- a/Makefile
+++ b/Makefile
@@ -31,7 +31,8 @@ docdir := ${prefix}/share/doc
 
 all: numad
 
-numad: numad.o -lpthread
+LDLIBS := -lpthread
+numad: numad.o
 
 AR ?= ar
 RANLIB ?= ranlib
-- 
1.9.2
